src/components/Quiz.js

```diff
@@ -1,186 +1,363 @@
-import React, { useState } from "react";
```

Comment:
There's a lot going on in this file and it's getting harder to read and understand the code - maybe you should split things up a bit? You could for example move the final "congratulations page" to it's own component in it's own file :)